apps/redirect/web/src/app/globals.css

```
@@ -0,0 +1,302 @@
+  align-items: center;
+  gap: 1rem;
+}
+/* Domain card header: let the (long) destination wrap instead of overflowing
+   and pushing the Remove button off the right edge on mobile. */
+.domain-head {
+  min-width: 0;
+  flex: 1;
+}
+.domain-head .dest {
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)